README.md

```diff
@@ -33,6 +33,7 @@ The following environment variables are required by the application container. V
 | COOKIE_PASSWORD                       | Redis cookie password      | yes      |                       |                             |
 | API_GATEWAY                           | Url of service API Gateway | no       | http://localhost:3001 |                             |
 | SESSION_TIMEOUT_IN_MINUTES            | Redis session timeout      | no       | 30                    |                             |
+| STATIC_CACHE_TIMEOUT_IN_MILLIS        | static file cache timeout  | no       | 54000 (15 minutes)    |                             |

```

Comment:
I feel like caching in dev is likely to cause frustration at some point. Should we default to no caching given that it's overridden from the helm chart/values for all non-dev environments?

Reply:
The caching is only at the browser level, so an 'F5' refresh or disabling the cache in browser dev tools is a simpler approach.

Reply:
Good point, I've had caching disabled for so long I forgot I'd done it